_sfp/ont-wo-mac.md

```diff
@@ -0,0 +1,84 @@
+---
+title: SFP with PON MAC and w/o PON MAC
+has_children: false
+nav_order: 2
+layout: default
+---
+
+PON technology unlike Ethernet are not P2P technologies but are one-to-many, and there are two device types: ONU (Optical Network Unit)/ONT (Optical Network Terminal) and OLT (Optical Line Terminal), both devices can be in SFP format[^tibit].
+
+The OLT provides an integrated access box for Passive Optical Networks. OLTs are typically chassis with one or more line cards inside, and on each line card there is one or more PON transceiver, usually in SFP format. Each line card is connected to a secondary switch provides line card aggregation to the Ethernet uplinks. OLTs are often a mixture of Layer 2 and Layer 3 switching with traffic shaping on a per-customer, per-service basis[^tibit].
+
+The communication within the SFP PON transceiver is neither MMI nor Ethernet, outside [SPF standards](/sfp-standard.md), but rather it is an *equivalent electrical symbols of optical transmission* (which is simply the input/output of the [BOSA](/bosa-tosa-rosa.md)) that for simplicity's sake we would call **PON RAW communication** (also referred to as SFP w/o PON MAC) and all the PON management part is left to the card lines. Each equivalent electrical symbol of optical transmission is a separate dialect, distinct from other dialects, furthermore, as you can easily guess, this communication is not standard and is not within the signalling standards ([^sfprate],[^sfprate2],[^sfpplusstandard]) but is compliant with MSA with some part of [^sfpstandard],[^sfpplusstandard],[^sfpplusmi]. This requires extreme compatibility between ONT and transreciver. This design choice to use RAW communication is made for several reasons:
+- *size*: the size of an OLT w/o PON MAC is very similar to that of anMMI nor Ethernet transceiver, and the size of an OLT with the integrated PON MAC part far exceeds that of the SFP format
+- *dissipative heating capacity*: the dissipative heating capacity of an OLT with PON MAC is higher than a normal transceiver, such as a 1 or 10 Gbps Ethernet link.
+- *duplication*: there is a double `MAC` → `MMI` conversion (`MMI` → `MAC` → `PHY` → `MAC` → `OLT CPU`)
+- *repairability*: since lasers often have a shorter lifetime than other ICs, it is good to be able to change only the tranreciver
```

Comment:
```suggestion
- *repairability*: since lasers often have a shorter lifetime than other ICs, it is good to be able to change only the transceiver
```